tests/plugins/split_to_column.test.ts

```diff
@@ -105,6 +106,13 @@ describe("Split text into columns", () => {
     expect(getGrid(model)).toEqual({ A1: "a", B1: "b", C1: "C1" });
   });
 
+  test("Localized splitted values are handled", () => {
```

Comment:
past tense of "split" is "split" :upside_down_face:  (not "splitted")

Reply:
splittationed